mono/mini/mini-ppc.c

```diff
@@ -5769,11 +5769,11 @@ mono_arch_emit_inst_for_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMetho
 	return NULL;
 }
 
-mgreg_t
+gssize
```

Comment:
This is dubious for x32 and amd64_32, will truncate.
In as much as we are looking at context for pointers, should be ok.
Not right if using debugger to look at registers in code doing 64bit integer math on in 32bit address space.

But we are mixing up host and target, and will require more change to straighten out, to support systems where registers and pointers are different size, x32 and arm64_32.